frappe/utils/password.py

```diff
@@ -62,7 +62,7 @@ def get_decrypted_password(doctype, name, fieldname="password", raise_exception=
 		return decrypt(result[0][0])
 
 	elif raise_exception:
-		frappe.throw(_("Password not found"), frappe.AuthenticationError)
+		frappe.throw(_(f"Password not found for {doctype} {name} {fieldname}"), frappe.AuthenticationError)
```

Comment:
F-strings can't work with translations. Use old style formatting with placeholders. 

https://frappeframework.com/docs/v14/user/en/translations#2-variables

Reply:
Thanks for the tips ankush and sorry about our disagreements a while ago, I shall behave more reasonably